chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/TestToolEntity.java

```diff
@@ -1,23 +1,30 @@
 package gov.healthit.chpl.entity;
 
 import java.util.Date;
+import java.util.List;
 
 import javax.persistence.Basic;
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
 import javax.persistence.Table;
 
+import gov.healthit.chpl.domain.CertificationResultTestTool;
+
 
 @Entity
 @Table(name = "test_tool")
 public class TestToolEntity {
 	
 	@Id 
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
-	@Basic( optional = false )
+	@Basic(optional = false)
+	@JoinColumn(name = "test_tool_id")
```

Comment:
The problem before was that I changed @Column(name = "blah") to @JoinColumn(name = "blah"). You can't see it in the window above because I didn't change @Column(name = "test_tool_id"), I just added @JoinColumn(name = "test_tool_id") which seems to be allowed in addition to @Column(name = "test_tool_id"). And yes, after hitting my head against the wall as many times as a zombie in The Walking Dead, somehow stuff still works: specifically, all unit tests pass, I can upload a CP, edit/update a CP, and I didn't notice any non-200 codes on other API calls.